db/migrate/20190722124603_rename_course_enrichment_qualifications.rb

```diff
@@ -0,0 +1,17 @@
+class RenameCourseEnrichmentQualifications < ActiveRecord::Migration[5.2]
+  def up
+    execute "UPDATE   course_enrichment
+                SET   json_data = Replace(json_data :: text, '\"Qualifications\":',
+                              '\"RequiredQualifications\":') ::
+                              jsonb
+              WHERE   json_data :: text LIKE '%\"Qualifications\":%';"
+  end
```

Comment:
I'm a little confused by this. Isn't the model now expecting `required_qualifications` to be the JSON field `Qualifications`? And isn't this replacing `"Qualifications"` with `"RequiredQualifications"`? Which would then break `required_qualifications`?

Reply:
I don't think the migration file is needed. Dan probably forgot about it.

ie, Data should be stored as `qualification` and then get hydrated as `required_qualification`

Reply:
Ok, so this needs to be removed, I think.